### PR TITLE
Use travis for windows builds

### PR DESCRIPTION
**Pull request by JonnyPtn**
Now travis has windows images, we don't need to use appveyor to cover all 3 platforms.

Also did a bit of tidying in the travis config

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,44 +11,42 @@ addons:
     - libvorbis-dev
     - g++
     - clang
+    - freeglut3-dev
```
Comment (eXpl0it3r) [inline .travis.yml]: Why did you add freeglut? Did you mean to add freetype?

Comment (JonnyPtn) [reply .travis.yml]: As discussed, it was a stab in the dark based on googling, and we've established libgl1-mesa-dev is better, so that's changed now
